registrations/tests.py

```diff
@@ -3945,10 +3958,14 @@ def test_send_outgoing_message_to_jembi_with_null_operator_id(self):
 
     @responses.activate
     def test_send_outgoing_message_to_jembi_invalid_user_id(self):
-        self.make_source_normaluser()
+        message_id = "BCGGJ3FVFUV"
+        jembi_url = "http://jembi/ws/rest/v1/helpdesk"
+        source = self.make_source_normaluser()
+        self.make_registration_for_jembi_helpdesk(source)
+        utils_tests.mock_request_to_jembi_api(jembi_url)
```

Comment:
I don't think we need all these extra bits in here (specifically the registration and the additional `mock_request_to_jembi_api`)

Reply:
okay will do